projects/cds/src/test.ts

```diff
@@ -0,0 +1,21 @@
+// This file is required by karma.conf.js and loads recursively all the .spec and framework files
```

Comment:
This file (and some other files) should not show up in the diff. Please merge the epic branch to this branch